internal/apps/dop/providers/project_report/metrics.go

```diff
@@ -105,6 +124,17 @@ func (i *IterationMetricFields) IsValid() bool {
 	return i.CalculatedAt.Day() == time.Now().Day()
 }
 
+func getIDs(a []uint64) string {
```

Comment:
duplicated function.

Reply:
thanks, this can indeed be optimized, you're right